### ISSUE TITLE
V12.2a CBUSH ouputs errors

### ISSUE THREAD

**Issue by numenic** [dr-bill-c/MYSTRAN#12]
I found some weired issues on CBUSH. Cf. attached test-case with one single cbush loaded under transversal loads :

[TC_single_bush.zip](https://github.com/dr-bill-c/MYSTRAN/files/6536751/TC_single_bush.zip)

Internal calculations are correct since on more complex models, displacements and reactions are still OK, whereas outpus keep wrong.

One can expect FZ (element)=1000lbf, MY (element)=-5000 lbf.in

F06 Engineering forces renders NULL values:

~~~~

                          E L E M E N T   E N G I N E E R I N G   F O R C E S
                              F O R   E L E M E N T   T Y P E   B U S H    
  Element      Force         Force         Force        Moment        Moment        Moment
     ID         XE            YE            ZE            XE            YE            ZE
        1  0.000000E+00  0.000000E+00  0.000000E+00  0.000000E+00  0.000000E+00  0.000000E+00
           ------------- ------------- ------------- ------------- ------------- ------------- 
 MAX* :    0.000000E+00  0.000000E+00  0.000000E+00  0.000000E+00  0.000000E+00  0.000000E+00
 MIN* :    0.000000E+00  0.000000E+00  0.000000E+00  0.000000E+00  0.000000E+00  0.000000E+00

 ABS* :    0.000000E+00  0.000000E+00  0.000000E+00  0.000000E+00  0.000000E+00  0.000000E+00
 *for output set
~~~~

NEU outputs strange results :

~~~~
BUSH Force ZE                 
    -1.056389E+03,    -1.056389E+03,     1.056389E+03,
       0,       0,       0,       0,       0,       0,       0,       0,       0,       0,
       0,       0,       0,       0,       0,       0,       0,       0,       0,       0,
       1,       1,       3,       8,
       0,       0,       1,
       1,    -1.056389E+03,
      -1,     0.          ,
       1,   51204,       1,
[...]
BUSH Moment YE                
     2.500000E+03,     2.500000E+03,     2.500000E+03,
       0,       0,       0,       0,       0,       0,       0,       0,       0,       0,
       0,       0,       0,       0,       0,       0,       0,       0,       0,       0,
       1,       1,       3,       8,
       0,       0,       1,
       1,     2.500000E+03,
      -1,     0.          ,
       1,   51206,       1,

~~~~

**Comment by numenic** [comment]
Additionally, here are the relevant sections of the NX-NASTRAN's F06

~~~~
                                             D I S P L A C E M E N T   V E C T O R
 
      POINT ID.   TYPE          T1             T2             T3             R1             R2             R3
             1      G      0.0            0.0            0.0            0.0            0.0            0.0
             2      G      0.0            6.036510E-01   0.0           -6.358734E-02   0.0            0.0
~~~~

~~~~
                                 F O R C E S   I N   B U S H   E L E M E N T S        ( C B U S H )
 
                  ELEMENT-ID        FORCE-X       FORCE-Y       FORCE-Z      MOMENT-X      MOMENT-Y      MOMENT-Z  
0                          1      0.0           0.0           1.000000E+03  0.0          -5.000000E+03  0.0
~~~~

~~~~
                             S T R E S S E S   I N   B U S H   E L E M E N T S        ( C B U S H )
 
                  ELEMENT-ID        STRESS-TX     STRESS-TY     STRESS-TZ    STRESS-RX     STRESS-RY     STRESS-RZ 
0                          1      0.0           0.0           1.000000E+03  0.0          -5.000000E+03  0.0
~~~~

**Comment by dr-bill-c** [comment]
I'll take a look at it

**Comment by dr-bill-c** [comment]
Hi numenic,
IYou must not be using the latest version 12.2 of MYSTRAN since I get the same answers as you posted for NX NASTRAN. See my zip file
[single_bush.zip](https://github.com/dr-bill-c/MYSTRAN/files/6620761/single_bush.zip)
 attached

**Comment by numenic** [comment]
Hello dr Bill,
Sorry for the late answer: I didn't have time to work on Mystran validation the last weeks.

Coming back to our issue, I'm completely puzzled. I downloaded the archive you proposed in your previous answer, I ran it with the latest V12.4, and I still have wrong values in the F06!

[single_bush_numenic.zip](https://github.com/dr-bill-c/MYSTRAN/files/6802471/single_bush_numenic.zip)

How is that possible? Could it be a compilation issue?
